src/tile-link/_tile-link-with-background.scss

```diff
@@ -53,5 +54,25 @@
       opacity: 1;
     }
   }
+
+  .tile-legal {
+    bottom: 1vw;
+    left: 1.5vw;
+    position: absolute;
+    color: $dark-grey;
+    font-size: 13px;
+    @include font($skytext-reg);
+    line-height: 1em;
+    width: 100%;
+
+    @include tablet {
+      bottom: 2vw;
```

Comment:
Don't suppose theres any way of 'variabalising' these across all the tile borders to ensure consistency of padding/margins?